src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -146,10 +147,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 });
             }
         } elseif ($options['multiple']) {
```

Comment:
What do you expect the html output to be when `widget` is set to `text` and
1- `expanded` is `true`
2- `expanded` and `multiple` are `false`
??